service/version_bundles.go

```diff
@@ -0,0 +1,47 @@
+package service
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func newVersionBundles() []versionbundle.Bundle {
+	return []versionbundle.Bundle{
+		{
+			Changelogs: []versionbundle.Changelog{},
```

Comment:
Changelog must not be empty. Same with starting the operator should fail.

Reply:
Done.